src/metabolic_ninja/worker/designer.py

```diff
@@ -79,6 +79,7 @@ def evaluate_diff_fva(designs, pathway, model, method):
     logger.info(
         f"Evaluating {len(designs) - 1} differential FVA surface points."
     )
+    table = designs.nth_panel(0)
```

Comment:
This should only get the diff FVA results for one point so it needs to be the right "panel" for each `design_result` in the below loop. (Based on discussion there seems to be a cameo bug. I will investigate.)